client/notifications/index.jsx

```diff
@@ -240,6 +240,15 @@ export class Notifications extends Component {
 					page( `/comment/${ siteId }/${ commentId }?action=edit` );
 				},
 			],
+			ANSWER_PROMPT: [
+				( store, { siteId, href } ) => {
+					this.props.checkToggle();
+					this.props.recordTracksEventAction( 'calypso_notifications_answer_prompt', {
```

Comment:
nice, I think we'll have to register this event in https://github.com/Automattic/tracks-events-registration after this is merged

Reply:
Cool, will do that...

Reply:
https://github.com/Automattic/tracks-events-registration/pull/1153